oauthenticator/github.py

```diff
@@ -54,6 +54,13 @@ def _github_client_secret_changed(self, name, old, new):
     client_secret_env = 'GITHUB_CLIENT_SECRET'
     login_handler = GitHubLoginHandler
     
+    github_to_nix_usernames_map = Dict(config=True, 
+        help="""Optional dict to remap github usernames to nix usernames.
```

Comment:
There's no need to use dedent on help strings. Any code that renders this help string should be responsible for dedent.